android/app/src/main/java/com/couchbase/todo/TasksFragment.java

```diff
@@ -253,7 +253,8 @@ public TaskAdapter(Context context, LiveQuery query) {
             if (thumbnail != null)
                 imageView.setImageBitmap(thumbnail);
             else
-                imageView.setImageDrawable(getResources().getDrawable(R.drawable.ic_camera_light));
+                //imageView.setImageDrawable(getResources().getDrawable(R.drawable.ic_camera_light));
+                imageView.setImageDrawable(getResources().getDrawable(R.drawable.ic_note));
```

Comment:
Sachin would like to change default icon of the task to note icon instead of the camera icon.